tests/regression/chk-1866.test.ts

```diff
@@ -0,0 +1,43 @@
+import { visitAndClearCookies, setup } from '../../utils'
+import { Accounts, SKUs } from '../../utils/constants'
+import { calculateShippingPreview } from '../../utils/shipping-actions'
+
+describe('CHK-1866', () => {
+  describe(`${Accounts.DEFAULT}`, () => {
+    beforeEach(() => {
+      visitAndClearCookies(Accounts.DEFAULT)
+    })
+
+    it('should the price shown in the shipping preview will be the same as shown in the cart summary', () => {
+      setup({
+        skus: [SKUs.ONLY_PICKUP_2_SLA_RJ, '75530389'],
```

Comment:
just a nitpick: maybe we should extract that `75530389` into a constant?

Reply:
Agree!

Reply:
Done!